src/serialization/toSerializableFormat.js

```diff
@@ -52,13 +49,16 @@ function transformSubstate(originalSubState) {
  * @returns {*}
  */
 export function toSerializableFormat(state) {
-  const accumulator = _.isArray(state) ? [] : {};
+  if (!_.isObjectLike(state)) {
+    return state;
+  }
 
-  return _.reduce(state, (serializableState, substate, subStateKey) => {
-    const serializableSubState = transformSubstate(substate);
+  const transformedState = transformSubstate(state);
```

Comment:
`transformXY` is just transforming state (array to object), nothing else. Serialization is recursive process until you reach ordinary value. And in that process, you need to transform something or sanitize it (`sanitizeElement` ...). I think naming is wrong. When I look now and before, I think this two process are better if they are divided.